src/app/(sidebar)/account/fund/page.tsx

```diff
@@ -1,5 +1,185 @@
 "use client";
 
+import { useState } from "react";
+import Link from "next/link";
+
+import {
+  Card,
+  Icon,
+  IconButton,
+  Input,
+  Text,
+  Button,
+} from "@stellar/design-system";
+
+import { shortenStellarAddress } from "@/helpers/shortenStellarAddress";
+import { validatePublicKey } from "@/helpers/validatePublicKey";
+import { useStore } from "@/store/useStore";
+
+import { AlertBox } from "@/components/AlertBox";
+
+import "../styles.scss";
+
+const callFriendBot = async ({
+  network,
+  publicKey,
+}: {
+  network: string;
+  publicKey: string;
+}) => {
+  const friendbotURL =
+    network === "futurenet"
+      ? "https://friendbot-futurenet.stellar.org"
+      : "https://friendbot.stellar.org";
+  const response = await fetch(friendbotURL + "/?addr=" + publicKey);
```

Comment:
Let's use a template string instead.

```suggestion
  const response = await fetch(`${friendbotURL}/?addr=${publicKey}`);
```